slsim/LsstSciencePipeline/lsst_science_pipeline.py

```diff
@@ -297,6 +298,9 @@ def lens_inejection_fast(
         is 3 arcsec).
     :param center_source_snr_threshold: SNR threshold for object
         detection in center box (default is 5).
+    :param false_positive: Boolean. If false, code assumes that the provided
+     population is a lens population. If True, code assumes that the provided
```

Comment:
this doctoring might need some more explanation or pointer to a specific function of what it does